### ISSUE TITLE
[Feature] Implement missing parameters for pushover, such as `ttl`

### ISSUE THREAD

**Issue by FrederikNJS** [prometheus/alertmanager#3454]
<!--

    Please do *NOT* ask usage questions in Github issues.

    If your issue is not a feature request or bug report use:
    https://groups.google.com/forum/#!forum/prometheus-users. If
    you are unsure whether you hit a bug, search and ask in the
    mailing list first.

    You can find more information at: https://prometheus.io/community/

-->

**What did you do?**

I used the pushover receiver, and tried to add the `ttl` parameter, but alertmanager didn't recognize the setting.

**What did you expect to see?**

I expected alertmanager to allow me to use any parameter that pushover supports: https://pushover.net/api

**What did you see instead? Under which circumstances?**

As mentioned above alertmanager crashes on launch, explaining that it doesn't recognize the `ttl` parameter.

**Environment**

* System information:

	$ uname -srm
	Linux 6.1.0-3-amd64 x86_64

* Alertmanager version:

	$ alertmanager --version
    alertmanager, version 0.25.0 (branch: HEAD, revision: 258fab7cdd551f2cf251ed0348f0ad7289aee789)
      build user:       root@abe866dd5717
      build date:       20221222-14:51:36
      go version:       go1.19.4
      platform:         linux/amd64

* Prometheus version:

	I'm not using prometheus, but victoria-metrics 1.91.4

* Alertmanager configuration file:
```
global:
  resolve_timeout: 5m
route:
  receiver: 'pushover'
  repeat_interval: 1h
  routes:
  - match:
      alertname: Watchdog
    receiver: "null"
  - match:
      alertname: InfoInhibitor
    receiver: "null"
receivers:
- name: "null"
- name: 'pushover'
  pushover_configs:
  - user_key: "{{ index . "pushover_key" }}"
    token: "{{ index . "token" }}"
    priority: '0'
    ttl: 3600
    send_resolved: false
```

* Prometheus configuration file:
```
Not really relevant
```

* Logs:
```
ts=2023-08-09T09:49:40.054Z caller=main.go:240 level=info msg="Starting Alertmanager" version="(version=0.25.0, branch=HEAD, revision=258fab7cdd551f2cf251ed0348f0ad7289aee789)"                                                                                                                                                                                             
ts=2023-08-09T09:49:40.054Z caller=main.go:241 level=info build_context="(go=go1.19.4, user=root@abe866dd5717, date=20221222-14:51:36)"                                                                                                                                                                                                                                      
ts=2023-08-09T09:49:40.350Z caller=coordinator.go:113 level=info component=configuration msg="Loading configuration file" file=/etc/alertmanager/config/alertmanager.yaml                                                                                                                                                                                                    
ts=2023-08-09T09:49:40.353Z caller=coordinator.go:118 level=error component=configuration msg="Loading configuration file failed" file=/etc/alertmanager/config/alertmanager.yaml err="yaml: unmarshal errors:\n  line 23: field ttl not found in type config.plain"
```

**Comment by heliapb** [comment]
Hi @gotjosh I would like to work on this issue, could you please assign it o me?

**Comment by chrko** [comment]
I think that has been implemented in #3474 and therefore can be closed.
